cli/pkg/release/gbm.go

```diff
@@ -221,20 +221,17 @@ func getChangeLog(dir string, gbmPr *gh.PullRequest) ([]byte, error) {
 	cl := []byte{}
 
 	if dir == "" {
```

Comment:
This `if dir == ""` logic block came from the original `gbm-cli` branch ([reference](https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/gbm-cli/cli/pkg/release/gbm.go#L164)), but I'm wondering if it's still needed. Currently, this logic would always choose this block as we're already passing dir as `""` directly. Addressing this would tackle this todo item and issue for handling the dir:

https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/blob/0073331ed6fab1391f8b14c010737b034487d949/cli/pkg/release/gbm.go#L163-L164

* https://github.com/orgs/wordpress-mobile/projects/208/views/1?pane=issue&itemId=42320242

Reply:
I'm not sure why I added the dir check in the original `gbm-cli` branch. I know it was trigger the fetch to the remote `githubusercontent` site but I'm not sure why we would need to do that if we have a cloned local version. 

It might had been for a status command or a preflight check but I'm not entirely sure we now need what's in the `if dir == ""` block. Instead we should be calling `getChangeLog` with the directory that holds the GBM code.

Reply:
Updated to call `getChangeLog` and `getReleaseNotes` with the `dir` value. I kept the original logic in case it's useful in the future.

Reply:
> I kept the original logic in case it's useful in the future.

Perfect